pyproject.toml

```
@@ -2,6 +2,11 @@
 requires = ["setuptools>=45", "wheel", "setuptools_scm>=8.1.0"]
 build-backend = "setuptools.build_meta"
 
+
+[tool.ruff.lint]
+select = ["E"]
```

I suggest to change it to "[extend-select](https://docs.astral.sh/ruff/settings/#lint_extend-select)" in order to keep the default values:
```suggestion
extend-select = ["E"]
```

Or what was your idea about this line?
If it leads to some discussion, we can remove this line in this PR and start an issue, which will lead to another PR.
That way we can merge this PR about an instrument.